main.go

```diff
@@ -119,7 +120,6 @@ func main() {
 		grpcfd.WithChainStreamInterceptor(),
 		grpcfd.WithChainUnaryInterceptor(),
 		grpc.WithDefaultCallOptions(
-			grpc.WaitForReady(true),
```

Comment:
@edwarnicke 
I think there will be a problem with this heal implementation - https://github.com/networkservicemesh/sdk/pull/1107
Because, we have only _one_ attempt to heal the connection. If this attempt fails - [link](https://github.com/networkservicemesh/sdk/pull/1107/files#diff-e4d431e9d4b1159f4966aee67576b5483344f2c03c5bd72ce34803bace4d0cc9R60) - we don't start a new `eventLoop` and therefore no new attempt will occur . 
Let's take a look at local `nsmgr` restart scenario. `nsmgr` will not be able to recreate instantly.
`grpc.WaitForReady` (or probably `WithBlock`) ensures that we wait for a new `nsmgr` to start and successfully complete the first healing attempt.
I think that's the reason why `ci` is red [here](https://github.com/networkservicemesh/sdk/pull/1107)...
Am I thinking right?